FpsCheck.py

```diff
@@ -243,7 +243,12 @@ def is_eTLD_Plus1(self, site):
         Returns:
             boolean with truth value dependent on value of get_public_suffix
         """
-        return self.etlds.get_public_suffix(site, strict=True) is not None
+        assert site is not None
+        site = site.lstrip("https://")
+        is_etld_plus1 = self.etlds.get_public_suffix(site, strict=True) == site
```

Comment:
It looks like we're supposed to use `get_sld` instead of this function: https://github.com/nexB/python-publicsuffix2/blob/4b2a771d621663dba214a66019a91a9989846a93/src/publicsuffix2/__init__.py#L279